platform/features/conductor/src/ConductorTelemetryDecorator.js

```diff
@@ -51,7 +51,7 @@ define(
                 request = request || {};
                 request.start = start;
                 request.end = end;
-                request.domain = domain;
+                request.domain = domain && domain.key;
```

Comment:
It seems to me that at this level, we're hiding invalid input errors that should actually cause the caller to receive an error; calling requestTelemetry without an array is invalid, and providing an array that contains items that are not objects is also invalid.  Likewise, the conductor should always be providing a real domain.